packages/koa-shopify-auth/src/auth/create-enable-cookies-redirect.ts

```diff
@@ -9,7 +9,9 @@ export default function createEnableCookiesRedirect(path: string) {
 
   return function topLevelOAuthRedirect(ctx: Context) {
     // This is to avoid a redirect loop if the app doesn't use verifyRequest or set the test cookie elsewhere.
-    ctx.cookies.set(TEST_COOKIE_NAME, '1');
+    ctx.cookies.set(TEST_COOKIE_NAME, '1', {
+      sameSite: 'none',
```

Comment:
secure: true
https://www.chromestatus.com/feature/5088147346030592

> (Cookies marked SameSite=None should also be marked Secure.)

Reply:
Also is this a safe change? Per https://github.com/Shopify/shopify_app/pull/851#issue-361762137 we may need to check the browser agent against a whitelist/blacklist.

> Additionally, SameSite=None (which we do not use today) is unfortunately interpreted incorrectly by specific browsers, so this PR handles such situations as well.

Reply:
The ideal solution in my opinion would be adding a config option that receives cookie options and letting the devs add the logic around this themselves and remove it whenever other browsers (Safari) decided to fix it!

```
createShopifyAuth({
    apiKey: <SHOPIFY_API_KEY>,
    secret: <SECRET>,
    // If browser != "Safari"
    cookieOptions: {
        sameSite: 'none',
        secure: true
    }
  }),
```